nautobot/project-static/css/base.css

```diff
@@ -347,6 +347,15 @@ ul.software-image-hierarchy {
     padding-left: 18px;
 }
 
+ul.tree-hierarchy {
+    list-style-type: none;
+    padding-left: 0px;
+}
+ul.tree-hierarchy ul {
+    list-style-type: "↳ ";
+    padding-left: 18px;
+}
```

Comment:
Is this coming from somewhere else? Can it be removed from there?

Reply:
Not really, as we currently use `location-hierarchy` and `software-image-hierarchy`. this is a more generic `hierarchy` class